operatorcourier/validate.py

```diff
@@ -75,6 +76,14 @@ def _bundle_validation(self, bundle):
         if self.ui_validate_io:
             validationDict[self.csvKey] &= self._type_validation(
                 bundleData, self.csvKey, self._ui_validation_io, True)
+        if validationDict[self.pkgsKey] and repository is not None:
+            packageName = bundleData['packages'][0]['packageName']
+            match = repository == packageName
+            if not match:
+                self._log_error('The packageName (%s) in bundle does not match '
+                                'repository name (%s) provided as command line argument.',
+                                packageName, repository)
+            validationDict[self.pkgsKey] &= match
```

Comment:
@jeremylinlin 
Why are we doing this bitwise and here? shouldn't we only define behavior in the case that they don't match?

Reply:
Thanks @kevinrizza, it makes sense. I have simplified the code.